vmm/src/vmm_config/instance_info.rs

```diff
@@ -206,6 +211,12 @@ impl Display for StartMicrovmError {
                     err_msg
                 )
             }
+            SeccompFilters(ref err) => {
+                let mut err_msg = format!("{:?}", err);
+                err_msg = err_msg.replace("\"", "");
+
+                write!(f, "Cannot build seccomp filters. {}", err_msg)
```

Comment:
ultra nit: ":" instead of "." to match the format of the errors from the jailer crate.